tests/test_server.py

```
@@ -1,6 +1,60 @@
+        f"/intermediate/{uuid.uuid4()}",
+        auth=BearerAuth(issue_client_access_token(analysis_id)),
+    )
+
+    try:
+        assert r.status_code == status.HTTP_502_BAD_GATEWAY
+        assert detail_of(r) == "Unexpected response from Hub (status code unknown): 'Test Error'."
+    finally:
+        test_client.app.dependency_overrides.pop(get_storage_client)
+
+
+def test_database_exception_handler(monkeypatch, test_client, analysis_id):
```

Ruff (0.15.2)

[warning] Unused function argument: `monkeypatch`

(ARG001)